.gitlab/new-e2e_testing/ubuntu.yml

```
@@ -15,6 +15,26 @@
     TEAM: agent-platform
     EXTRA_PARAMS: --osversion $E2E_OSVERS --platform $E2E_PLATFORM --cws-supported-osversion $E2E_CWS_SUPPORTED_OSVERS --major-version $AGENT_MAJOR_VERSION --arch $E2E_ARCH --flavor $FLAVOR
 
+.new-e2e_script_upgrade:
+  variables:
+    TARGETS: ./tests/agent-platform/upgrade
+    TEAM: agent-platform
+    EXTRA_PARAMS: --osversion $E2E_OSVERS --platform $E2E_PLATFORM --cws-supported-osversion $E2E_CWS_SUPPORTED_OSVERS --arch $E2E_ARCH --flavor $FLAVOR
+  parallel:
+    matrix:
+      - START_MAJOR_VERSION: [5,6]
+        AGENT_MAJOR_VERSION: [6, 7]
+      - START_MAJOR_VERSION: [7]
+        AGENT_MAJOR_VERSION: [7]
+  script:
+    - export DATADOG_AGENT_API_KEY=$(aws ssm get-parameter --region us-east-1 --name ci.agent-linux-install-script.datadog_api_key --with-decryption --query "Parameter.Value" --out text)
+    - inv -e new-e2e-tests.run --targets $TARGETS -c ddagent:fullImagePath=669783387624.dkr.ecr.us-east-1.amazonaws.com/agent:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA} -c ddagent:clusterAgentFullImagePath=669783387624.dkr.ecr.us-east-1.amazonaws.com/cluster-agent:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA} -c dddogstatsd:fullImagePath=669783387624.dkr.ecr.us-east-1.amazonaws.com/dogstatsd:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA} -c ddagent:imagePullRegistry=669783387624.dkr.ecr.us-east-1.amazonaws.com -c ddagent:imagePullUsername=AWS -c ddagent:imagePullPassword=$(aws ecr get-login-password) --junit-tar "junit-${CI_JOB_NAME}.tgz" ${EXTRA_PARAMS} --src-agent-version $START_MAJOR_VERSION --dest-agent-version $AGENT_MAJOR_VERSION
```

```suggestion
    - inv -e new-e2e-tests.run --targets $TARGETS ${EXTRA_PARAMS} --src-agent-version $START_MAJOR_VERSION --dest-agent-version $AGENT_MAJOR_VERSION
```
I think we do not need all the docker related variables here